EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||adopalstroy.com^
```

Comment:
Why without `$third-party`?

Reply:
Why not?

Reply:
You can use it in some cases, but it is safer to add using a modificator. We have a long story with selling domains and broken sites.

Reply:
Ok I will change.